beacon-chain/powchain/service.go

```diff
@@ -321,7 +323,7 @@ func (w *Web3Service) ProcessLog(depositLog gethTypes.Log) {
 // the ETH1.0 chain by trying to ascertain which participant deposited
 // in the contract.
 func (w *Web3Service) ProcessDepositLog(depositLog gethTypes.Log) {
-	merkleRoot, depositData, merkleTreeIndex, _, err := contracts.UnpackDepositLogData(depositLog.Data)
+	_, depositData, merkleTreeIndex, _, err := contracts.UnpackDepositLogData(depositLog.Data)
```

Comment:
We no longer need the Merkle root as deposits instead have their proofs generated on demand by the Merkle trie in the RPC service